packages/babel-plugin/src/__fixtures__/template-literal/output.js

```diff
@@ -0,0 +1,17 @@
+const A = styled.div({
+  'name': 'code__A',
+  'hash': 'cpLxln',
+  'nodes': [[13, 'color'], [14, 'green'], [13, 'background'], [22, 'green']]
```

Comment:
This maybe won't matter, but what if we built a hash in memory of all the values and imported them per file, babel helper style? I think it might help dedupe the overall output. I know gzip ends up squashing it for the most part, but there still is a cost...